frontend/src/components/pages/LogRecords.tsx

```diff
@@ -60,9 +67,81 @@ const LogRecords = (): React.ReactElement => {
   const [logRecords, setLogRecords] = useState<LogRecord[]>([]);
   console.log(logRecords);
 
+  const [email, setEmail] = useState<string>("");
+  const [invitedEmail, setInvitedEmail] = useState<string>("");
+  const [invitedFirstName, setInvitedFirstName] = useState<string>("");
+  const [invitedLastName, setInvitedLastName] = useState<string>("");
+
+  const [userRole, setUserRole] = useState<string>("");
+  const [isInvited, setIsInvited] = useState<boolean>(false);
+
+  const inviteUser = async () => {
+    await commonApiClient.inviteUser(
+      email,
+      userRole,
+      invitedFirstName,
+      invitedLastName,
+    );
+  };
+  const checkInvitedUser = async () => {
+    const res = await commonApiClient.isUserInvited(invitedEmail);
+    setIsInvited(res);
+  };
   return (
     <div className="page-container">
       <NavigationBar />
+      <Box padding="20px">
```

Comment:
since you have a separate page for inviting users, can you remove these changes :D